src/Microsoft.Health.Fhir.Core/Features/Search/Registry/SearchParameterStatusManager.cs

```diff
@@ -138,7 +138,7 @@ public async Task UpdateSearchParameterStatusAsync(IReadOnlyCollection<string> s
 
             foreach (string uri in searchParameterUris)
             {
-                _logger.LogTrace("Setting the search parameter status of '{Uri}' to '{NewStatus}'", uri, status.ToString());
+                _logger.LogInformation("Setting the search parameter status of '{Uri}' to '{NewStatus}'", uri, status.ToString());
```

Comment:
qq, can there be a lot of these? is this 10s or 100s of log entries?

Reply:
I would say 10s, it is only called when a customer search parameter is added, updated, or deleted.